auth0/structure_auth0_connection.go

```diff
@@ -368,6 +368,7 @@ func flattenConnectionOptionsEmail(options *management.ConnectionOptionsEmail) (
 		"brute_force_protection":   options.GetBruteForceProtection(),
 		"set_user_root_attributes": options.GetSetUserAttributes(),
 		"non_persistent_attrs":     options.GetNonPersistentAttrs(),
+		"auth_params":              options.AuthParams,
```

Comment:
This should probably not be here if we're checking it on line 389?

Reply:
I may be mistaken, but during testing I thought I needed this or else tests would fail. Will double check because it does appear odd.